src/SFA.DAS.Assessor.Functions/Functions/Ofs/OfsImportFunction.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Assessor.Functions.Domain.Assessors.Interfaces;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Assessor.Functions.Functions.Ofs
+{
+    public class OfsImportFunction
+    {
+        private readonly IOfsImportCommand _command;
+
+        public OfsImportFunction(IOfsImportCommand command)
+        {
+            _command = command;
+        }
+
+        [FunctionName("OfsImport")]
+        public async Task Run([TimerTrigger("%FunctionsOptions:OfsImportOptions:Schedule%", RunOnStartup = false)] TimerInfo myTimer, ILogger log)
+        {
+            try
+            {
+                if (myTimer.IsPastDue)
+                {
+                    log.LogInformation($"OfsImport has started later than scheduled");
```

Comment:
It might be helpful to also mentioned the scheduled time in this log entry. The `TimerInfo` class should have a property that knows its own scheduled time, which we can use to log something like
```suggestion
                    log.LogInformation($"OfsImport has started later than the scheduled time of {myTimer.ScheduledTimeOrWhatever}");
```